app/assets/stylesheets/iframe.css.scss

```diff
@@ -0,0 +1,57 @@
+body.iframe {
+
+  #application-container {
+    width: 100%;
+  }
+
+  #upper-corner-console,
+  #top-nav-frame {
+    display: none;
+  }
+
+  #application-body {
+    margin: 5px;
+    min-height: 250px;
+  }
+  #page-heading {
+    padding: 0;
+  }
+
+  .login-options {
+    width: 100%;
+    float: none;
+  }
+
+  #login-explanation {
+    padding: 0;
+    width: 100%;
+  }
+  // login page
+  .login-options {
+    .or-bar {
+      width: 100%;
+      .or-text {
+        display: inline;
+        margin-left: auto;
+        margin-rigth: auto;
+      }
+    }
+    .password-login {
+      input[type='text'], input[type='password'] {
+        width: 100%;
+      }
+      .password-actions .sign-up {
+        float: inherit;
+        margin-left: 35px;
+      }
+    }
+
+  }
+
+  //signup page
+  #signup-alternatives {
+    float: inherit;
+    width: 80%;
+    margin: 0 auto 20px auto;
+  }
+}
```

Comment:
Styling could use further tweaks.  For instance, we might want to hide the footer content with the terms and about links.